explorer-api/src/country_statistics/mod.rs

```diff
@@ -51,17 +52,35 @@ impl CountryStatistics {
 
```

Comment:
Thid actually refers to line 46 - there's another unnecessary Arc clone there : )

Reply:
I just really want to increment the reference count - feels really wrong not to #cpp